app/src/play/java/com/amaze/filemanager/activities/MainActivity.java

```diff
@@ -166,441 +166,460 @@
-        SmbConnectionListener, DataChangeListener, BookmarkCallback,
-        SearchWorkerFragment.HelperCallbacks, CloudConnectionCallbacks,
-        LoaderManager.LoaderCallbacks<Cursor> {
-
-    public static final Pattern DIR_SEPARATOR = Pattern.compile("/");
-    public static final String TAG_ASYNC_HELPER = "async_helper";
-
-    /* Request code used to invoke sign in user interactions. */
-    static final int RC_SIGN_IN = 0;
-
-    private DataUtils dataUtils = DataUtils.getInstance();
-
-    public DrawerLayout mDrawerLayout;
-    public ListView mDrawerList;
-    public ScrimInsetsRelativeLayout mDrawerLinear;
-    public String path = "", launchPath;
-    public ArrayList<BaseFile> COPY_PATH = null, MOVE_PATH = null;
-    public FrameLayout frameLayout;
-    public boolean mReturnIntent = false;
-    public boolean useGridView, openzip = false;
-    public boolean mRingtonePickerIntent = false, colourednavigation = false;
-    public int skinStatusBar;
-
-    public volatile int storage_count = 0; // number of storage available (internal/external/otg etc)
-
-    public FloatingActionMenu floatingActionButton;
-    public LinearLayout pathbar;
-    public FrameLayout buttonBarFrame;
-    public boolean isDrawerLocked = false;
-
-    public DrawerAdapter adapter;
-    public MainActivityHelper mainActivityHelper;
-
-    public int operation = -1;
-    public ArrayList<BaseFile> oparrayList;
-    public ArrayList<ArrayList<BaseFile>> oparrayListList;
-
-    // oppathe - the path at which certain operation needs to be performed
-    // oppathe1 - the new path which user wants to create/modify
-    // oppathList - the paths at which certain operation needs to be performed (pairs with oparrayList)
-    public String oppathe, oppathe1;
-    public ArrayList<String> oppatheList;
-    public RelativeLayout drawerHeaderParent;
-
-    /**
-     * @deprecated use getCurrentMainFragment()
-     */
-    public MainFragment mainFragment;
-
-    public static final String KEY_PREF_OTG = "uri_usb_otg";
-
-    private static final int image_selector_request_code = 31;
-
-    private AppBar appbar;
-    //private HistoryManager history, grid;
-    private Futils utils;
-    private MainActivity mainActivity = this;
-    private Context con = this;
-    private String zippath;
-    private FragmentTransaction pending_fragmentTransaction;
-    private String pendingPath;
-    private boolean openProcesses = false;
-    private int hidemode;
-    private MaterialDialog materialDialog;
-    private String newPath = null;
-    private boolean backPressedToExitOnce = false;
-    private Toast toast = null;
-    private ActionBarDrawerToggle mDrawerToggle;
-    private Intent intent;
-    private GoogleApiClient mGoogleApiClient;
-    private View drawerHeaderLayout;
-    private View drawerHeaderView, indicator_layout;
-    private RoundedImageView drawerProfilePic;
-    private ImageLoader mImageLoader;
+/*
```

Comment:
This is duplicated, check line 1, Android Studio collapses the comment automatically.